webclient/src/core.js

```diff
@@ -10,18 +10,16 @@ let navigatum;
 const cachedFetch = (() => ({
   fetch: function (url, options) {
     return new Promise((resolve) => {
+      // Add language query param to the request
+      url += (url.indexOf("?") > 0 ? "&lang=" : "?lang=") + (localStorage.getItem("lang") || "de");
```

Comment:
No strong oppinion, but this seems a bit eazier to parse. What do you think?
```suggestion
      const lang = localStorage.getItem("lang") || "de";
      url += url.includes("?") ? `&lang=${lang}` : `?lang=${lang}`;
```

Reply:
I've updated it, but I'd still prefer the string concatenation. Would you agree on the current version?
```js
      const lang = document.documentElement.lang;
      url += (url.includes("?") ? "&lang=" : "?lang=") + lang;

```

Reply:
Yes, we could even inline lang, since the reason I did extract it is no longer present